pkg/scheduler/util/utils.go

```diff
@@ -210,3 +213,42 @@ func (l *SortableList) Sort() {
 func HigherPriorityPod(pod1, pod2 interface{}) bool {
 	return GetPodPriority(pod1.(*v1.Pod)) > GetPodPriority(pod2.(*v1.Pod))
 }
+
+// FilterPodsWithPDBViolation groups the given "pods" into two groups of "violatingPods"
```

Comment:
Could we add some tests for this?

Reply:
When remove PDBs, this method is not needed in utils (which belongs to preemption logic)